# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../musl/pkg/mdadm/0006-mdadm-musl-execl-missing-sentinel.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

error: missing sentinel in function call [-Werror=format]

--- a/util.c	2013-12-16 19:20:10.501872646 +0100
+++ b/util.c	2013-12-16 19:20:32.118396213 +0100
@@ -1656,11 +1656,11 @@
 				if (__offroot) {
 					execl(paths[i], "mdmon", "--offroot",
 					      devnum2devname(devnum),
-					      NULL);
+					      (char *)NULL);
 				} else {
 					execl(paths[i], "mdmon",
 					      devnum2devname(devnum),
-					      NULL);
+					      (char *)NULL);
 				}
 			}
 		exit(1);
--- a/Monitor.c	2013-12-16 19:21:05.623164042 +0100
+++ b/Monitor.c	2013-12-16 19:23:33.718570949 +0100
@@ -361,7 +361,7 @@
 			break;
 		case 0:
 			execl(info->alert_cmd, info->alert_cmd,
-			      event, dev, disc, NULL);
+			      event, dev, disc, (char *)NULL);
 			exit(2);
 		}
 	}
